website/tables/aws/aws_wellarchitected_lenses.md

```diff
@@ -0,0 +1,31 @@
+# Table: aws_wellarchitected_lenses
+
+This table shows data for AWS Well-Architected Lenses.
+
+https://docs.aws.amazon.com/wellarchitected/latest/APIReference/API_Lens.html
+
+The primary key for this table is **arn**.
+
+## Columns
+
+| Name          | Type          |
+| ------------- | ------------- |
+|_cq_source_name|`utf8`|
+|_cq_sync_time|`timestamp[us, tz=UTC]`|
+|_cq_id|`uuid`|
+|_cq_parent_id|`uuid`|
+|account_id|`utf8`|
```

Comment:
Not sure about this in the docs it seems like the AWS lens' don't seem to have an accountID, in them... do we need to add a `request_account_id` here and make it a PK?

`arn:aws:wellarchitected:us-east-1::lens/serverless`
https://docs.aws.amazon.com/wellarchitected/latest/APIReference/API_GetLens.html

Reply:
I'd rather do `account_id` as PK (+ region), WDYT?

Reply:
Added acc + region to PK in c97c9f19dfc12fd34e2f7d148a24c194f1b92911